### DISCUSSION TITLE
Disabling In Dimension

### DISCUSSION THREAD

**Discussion by Yoskio** [Luke100000/minecraft-comes-alive#331] [Q&A] [answered]
Hello im just wondering if i can disable the villagers in specific dimensions that also have villages. Mainly Blue Skies because they have custom villagers that also do extra stuff like give you info in the blue book. With MCA it doesnt seem to work too well. so i ask is there anyway to limit the MCA villagers to the Overworld?

**Comment by CDAGaming** [comment, marked as answer]
Added in 7.3.19
